src/python/pants/backend/shell/target_types.py

```diff
@@ -324,6 +335,7 @@ class ShellCommandTarget(Target):
         ShellCommandSourcesField,
         ShellCommandTimeoutField,
         ShellCommandToolsField,
+        ShellCommandExtraEnvVarsField,
```

Comment:
Added this to the end but looks like it should maybe be put in alphabetical order? 🤔

Reply:
I'm one in favor of sorting lists alphabetically, however here I think the order affects the order the fields are presented in the documentation so that is worth considering too.